e2e_deployment_files/connectivity_deployment.template.json

```diff
@@ -649,9 +649,6 @@
               "testStartDelay": {
                 "value": "<TestStartDelay>"
               },
-              "transportType": {
```

Comment:
why removed? and btw I found that this key casing is not matched.  In C2DMessageTester settings.cs, it started with capital T.  Please help to create a bug and let Dylan aware of it.

Reply:
I noticed that transportType is not used by a c2dMessageSender in send mode. I talked with Dylan to confirm and we decided to remove it.

Reply:
But this is not great in current settings class, we should not have a value for transport type in settings if it runs in sender mode.  Maybe create a bug to track.

Reply:
Are you saying the desired behavior is to throw an exception in the case where you use transportType and are a sender?

Reply:
No.  If you are running a sender, the code to extract transport type should not even be run.